script.py

```diff
@@ -27,54 +29,64 @@ def generate_token(client_id, client_secret):
         print('Invalid credentials')
 
 
-def request_vehicles(token):
-    return requests.get(f'{osu_API}v1/beaverbus/vehicles', headers={
-        'Authorization': f'Bearer {token}'
-    })
-
-
-def request_vehicle_by_ID(token, ID):
-    return requests.get(f'{osu_API}v1/beaverbus/vehicles/{ID}', headers={
-        'Authorization': f'Bearer {token}'
-    })
+def request_vehicles(*arg):
+    if len(arg) == 1:
+        return requests.get(f'{osu_API}v1/beaverbus/vehicles', headers={
+            'Authorization': f'Bearer {arg[0]}'
+        })
+    if len(arg) == 2:
+        return requests.get(f'{osu_API}v1/beaverbus/vehicles/{arg[1]}',
```

Comment:
`f'{osu_API}v1/beaverbus/vehicles` is shared by both of the `if` block. You should reuse it in this function.